src/pre_defined_classes/mathematical_program/medium/q1/BMI.java

```diff
@@ -0,0 +1,16 @@
+package pre_defined_classes.mathematical_program.medium.q1;
+
+/*
+Make a program that calculates and prints the body mass index (BMI) of a person given their weight in kilograms and
+height in metres. The formula for BMI is: weight / (height)^2
+ */
+
+public class BMI {
+    public static void main(String[] args) {
+        double weight = 70.5;
+        double height = 1.63;
+        double bmi = weight / Math.pow(height, 2);
+        System.out.print("The BMI of a person that weighs " + weight + " kilograms");
+        System.out.print(" and is " + height + " metres tall is: " + bmi);
```

Comment:
It's being weird because it expects all output to end with a newline, essentially. So the final print statement is expected to by a `println()`. You can continue testing if you make this second statement a println, but we'll patch this bug tonight - make an issue for me called "Patch print vs println regex bug" (for the practice)

Reply:
Ok, I just changed the test code to end with a println for now